### PR TITLE
Create ui-c8y-1018-503-100-added-c-8-y-dropdown-direction-directive-to-c-8-y.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-56532](https://cumulocity.atlassian.net/browse/MTM-56532)
Original PR: [6292](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6292)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1018-503-100-added-c-8-y-dropdown-direction-directive-to-c-8-y.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Added c8yDropdownDirection directive to c8y-li-action to fix action cutoff (#5213) [GRAFT][release/y2024] (#6292)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-100-added-c-8-y-dropdown-direction-directive-to-c-8-y.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fixed action menu cutoff in list items
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-100-added-c-8-y-dropdown-direction-directive-to-c-8-y.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-56532
+version: 1018.503.100
+---
+Added c8yDropdownDirection directive to c8y-li-action to fix action cutoff (#5213) [GRAFT][release/y2024] (#6292)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-100-added-c-8-y-dropdown-direction-directive-to-c-8-y.md:17]: The Claude AI suggested to use the following body:
```suggestion
In some cases, the action menu in list items could be cut off and not fully visible. To address this issue, the c8yDropdownDirection directive has been added to the c8y-li-action component. With this change, the action menu now properly adjusts its position based on the available space, ensuring that all actions are fully accessible and visible to users. This improvement enhances the usability of action menus within list items across the application.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by eniosultan** [comment]
I forgot to add the no change note label.

**Pull request closed without merging**